mlxtend/classifier/stacking_cv_classification.py

```diff
@@ -0,0 +1,270 @@
+        If True, the meta-classifier will be trained both on the predictions
+        of the original classifiers and the original dataset.
+        If False, the meta-classifier will be trained only on the predictions
+        of the original classifiers.
+    stratify : bool (default: True)
+        If True, the cross-validation technique used for fitting the classifier
+        will be Stratified K-Fold.
+        If False, the cross-validation technique used will be Regular K-Fold.
+        It is highly recommended to use Stratified K-Fold.
+    verbose : int, optional (default=0)
+        Controls the verbosity of the building process.
+        - `verbose=0` (default): Prints nothing
+        - `verbose=1`: Prints the number & name of the regressor being fitted
+                       and which fold is currently being used for fitting
+        - `verbose=2`: Prints info about the parameters of the
+                       regressor being fitted
+        - `verbose>2`: Changes `verbose` param of the underlying regressor to
+           self.verbose - 2
+
+    Attributes
+    ----------
+    clfs_ : list, shape=[n_classifiers]
+        Fitted classifiers (clones of the original classifiers)
+    meta_clf_ : estimator
+        Fitted meta-classifier (clone of the original meta-estimator)
+
+    """
+    def __init__(self, classifiers, meta_classifier,
+                 use_probas=True, n_folds=2,
+                 use_features_in_secondary=False,
+                 stratify=True, verbose=0):
+
+        self.classifiers = classifiers
+        self.meta_classifier = meta_classifier
+        self.named_classifiers = {key: value for
+                                  key, value in
+                                  _name_estimators(classifiers)}
+        self.named_meta_classifier = {'meta-%s' % key: value for
+                                      key, value in
+                                      _name_estimators([meta_classifier])}
+        self.use_probas = use_probas
+        self.verbose = verbose
+        self.n_folds = n_folds
+        self.use_features_in_secondary = use_features_in_secondary
+        self.stratify = stratify
+
+    def fit(self, X, y):
+        """ Fit ensemble classifers and the meta-classifier.
+
+        Parameters
+        ----------
+        X : {array-like, sparse matrix}, shape = [n_samples, n_features]
+            Training vectors, where n_samples is the number of samples and
+            n_features is the number of features.
+        y : array-like, shape = [n_samples]
+            Target values.
+
+        Returns
+        -------
+        self : object
+
+        """
+        self.clfs_ = [clone(clf) for clf in self.classifiers]
+        self.meta_clf_ = clone(self.meta_classifier)
+        if self.verbose > 0:
+            print("Fitting %d classifiers..." % (len(self.classifiers)))
+
+        if self.stratify:
+            skf = StratifiedKFold(y, n_folds=self.n_folds)
```

Comment:
Good point! I think we should definitely set `shuffle=True` by default (and add a shuffle `__init__` parameter). Otherwise this may cause weird results if people didn't shuffle their datasets.